### PR TITLE
Changes to README and Graphs 

### PR DESCRIPTION
**Pull request by jaceybronte**
@gwaybio 
Changed --force to --yes in README, adjusted size and labels of some graphs to improve readability.

### CODE REVIEW

**Review by gwaybio — CHANGES_REQUESTED** [review]
Thanks for the bite-sized and well documented PR! Here are a couple comments to address prior to merging:

1. Please also add the updated figures to this PR
2. Please also add an `nbconverted` `.py` file. See the [nbconvert docs](https://nbconvert.readthedocs.io/en/latest/usage.html) for details. Having a `.py` file alongside the `.ipynb` file makes review easier (see this folder: https://github.com/WayScience/gene_dependency_representations/tree/main/1.data-exploration/scripts/nbconverted)
3. You might benefit from adding a comment to the conda command specifying the conda version (since the `--yes` flag is so new)
4. I see the commit author is different than your GitHub account. You might be running into the issue described here: https://docs.github.com/en/pull-requests/committing-changes-to-your-project/troubleshooting-commits/why-are-my-commits-linked-to-the-wrong-user which should be a quick fix globally on maple

**Comment by jaceybronte** [comment]
@gwaybio I think all the changes have been made!

File: 1.data-exploration/1.model_data_visualization.ipynb [hunk 1]
```diff
@@ -701,10 +701,13 @@
...
     "cancer_types_bar = (\n",
     "    gg.ggplot(model_df, gg.aes(x=\"OncotreePrimaryDisease\"))\n",
-    "    + gg.geom_bar()\n",
+    "    + gg.geom_bar(width = 0.5, position = position_dodge2(padding = 0.5))\n",
     "    + gg.coord_flip()\n",
+    "    + gg.xlab(\"Primary Disease\")\n",
+    "    + gg.ylab(\"Count\")\n",
     "    + gg.ggtitle(\"Distribution of cancer types\")\n",
     "    + gg.theme_bw()\n",
+    "    + theme(figure_size = (16, 12))\n",
```
Comment (gwaybio) [inline 1.data-exploration/1.model_data_visualization.ipynb]:
```suggestion
    "    + gg.theme(figure_size = (16, 12))\n",
```

For consistency.

FWIW, the only reason this works is the imports are a bit wonky:

```python
import plotnine as gg
from plotnine import *  # This line prob should be removed
```

File: 1.data-exploration/1.model_data_visualization.ipynb [hunk 2]
```diff
@@ -802,6 +805,7 @@
     "    gg.ggplot(model_df, gg.aes(x=\"Age\"))\n",
     "    + gg.geom_density()\n",
     "    + gg.geom_vline(xintercept=adult_threshold, linetype=\"dashed\", color=\"red\")\n",
+    "    + annotate(geom=\"text\", x=30, y = 0.0045, label=\"Adult Threshold\", color=\"red\")    \n",
```
Comment (gwaybio) [inline 1.data-exploration/1.model_data_visualization.ipynb]:
```suggestion
    "    + gg.annotate(geom=\"text\", x=30, y = 0.0045, label=\"Adult Threshold\", color=\"red\")    \n",
```

I like how this looks on the figure!

File: 1.data-exploration/1.model_data_visualization.ipynb [hunk 3]
```diff
@@ -1407,6 +1411,7 @@
     "    + gg.ylab(\"count\")\n",
     "    + gg.xlab(\"cancer type\")\n",
     "    + gg.theme_bw()\n",
+    "    + theme(figure_size = (14, 6))\n",
```
Comment (gwaybio) [inline 1.data-exploration/1.model_data_visualization.ipynb]:
```suggestion
    "    + gg.theme(figure_size = (14, 6))\n",
```

File: 1.data-exploration/1.model_data_visualization.ipynb [hunk 4]
```diff
@@ -1841,6 +1846,7 @@
     "    + gg.ylab(\"count\")\n",
     "    + gg.xlab(\"cancer type\")\n",
     "    + gg.theme_bw()\n",
+    "    + theme(figure_size = (14, 10))\n",
```
Comment (gwaybio) [inline 1.data-exploration/1.model_data_visualization.ipynb]:
```suggestion
    "    + gg.theme(figure_size = (14, 10))\n",
```

**Review by gwaybio — APPROVED** [review]
Super nitpick, but I noticed `plotnine` wasn't being called consistently.

Also, FWIW, we use plotnine (and seaborn/matplotlib) for quick data exploration, but `ggplot2` for publication quality figures

After you address the plotnine consistency, please feel free to squash and merge! (no need for me [or anyone else] to review again after you receive one approval ✅ )

**Pull request merged**